pkg/apis/policy/types.go

```diff
@@ -209,7 +209,7 @@ type PodSecurityPolicySpec struct {
 	// +optional
 	AllowedFlexVolumes []AllowedFlexVolume
 	// AllowedCSIDrivers is a whitelist of inline CSI drivers that must be explicitly set to be embedded within a pod spec.
-	// An empty value means no CSI drivers can run inline within a pod spec.
+	// An empty value indicates that any CSI driver can be used for inline ephemeral volumes.
```

Comment:
please add "This is an alpha field, and is only honored if the API server enables the CSIInlineVolume feature gate." (here and in both external versions, and update swagger/doc files)